src/main/webapp/step3-startup.js

```diff
@@ -0,0 +1,36 @@
+import Step3Renderer from './renderers/Step3Renderer.js';
```

Comment:
Can't you just use the startup.js and Renderer files that we already have? They seem very similar

Reply:
The renderer class should be different anyways, because it generates different HTML code.
Now, I also advocate to have different startup pages, because on the future we will need different functionality for each page (like for example, handle click of the next button, adding events, etc.)

Reply:
Sgtm